### PR TITLE
Project 1: create animals table.

### PR DESCRIPTION
**Pull request by joramot**
**### Project requirements**

- Used the template to generate the repo.
- Created a database named vet_clinic.
- Created a table named animals whit the columns.
- Inserted the data in the animals table.
- Wrote and executed queries for each quest.
- Took a screenshot of the results of your queries.

1 Find all animals whose name ends in mon
2 List the name of all animals born between 2016 and 2019
3 List the name of all animals that are neutered and have less than 3 escape attempts
4 List the date of birth of all animals named either Agumon or Pikachu
5 List name and escape attempts of animals that weigh more than 10 5kg
6 Find all animals that are neutered
7 Find all animals not named Gabumon
8 Find all animals with a weight between 10 4kg and 17 3kg (including the animals with the weights that equals precisely 10 4kg or 17 3kg)

### CODE REVIEW

**Review by skyv26 — CHANGES_REQUESTED** [review]
## Hi @joramot, 

Good job so far 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## STATUS: REQUIRE CHANGES 🔴 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 

## Required Changes ♻️
- [ ] Dear @joramot, you did a super cool job by implementing the things as per the requirement, I saw that your results are accurately matching and everything is perfect except one thing, which is the only cause that you have to re-submit your project for the review. I noticed that you have used the `Microverse` **README** template but forgot to alter the documentation with relevant information, as instructed at the very beginning of the journey into *Microverse*. Kindly remove the unnecessary section from your `README` file, and add everything relevant and related to this project.

- [ ] It would be nice if instead of adding the images in your repository, you could either put all the images inside the folder **OR** you can remove them because your screenshots can be uploaded through the github `attach` files feature.
 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

#### Cheers, and Happy coding!👏👏👏

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -100,19 +82,16 @@ After you're finished please remove all the comments and instructions!
...
-- **[key_feature_3]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+> coming soon...
```
Comment (TadesseDev) [inline README.md]: - [ ] Good job updating the readme file as suggested by the previous reviewer, but please consider removing the `Live demo` section from the readme file. It is only required when you have a project that is deployed. This is a mandatory requirement for all projects.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @joramot 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

199733072-bdacca19-8df9-43a7-be93-3aff2f1c15c8

 ## Highlights
 - [x] Good job making most of the changes requested by the previous reviewer. 🙏


 ## Required Changes ♻️
#### While you did a great job updating your readme as suggested earlier, Please notice the following facts.
- [ ] Please notice the section in your readme which starts with `>` are directions/guidelines from microverse to help you understand what to write in that section and should not be part of your readme file. Please consider removing this and all the like contents from the readme file. Please remove the `> Give credit to everyone who inspired your codebase.` part in your Acknowledgement section. 
- [ ] Also, Your licence section has section description text which shouldn't be part of your readme file.

 _Check the comments under the review._

## Optional suggestions

- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @joramot, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

 

## Highlights
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**